JSDemos/Demos/TreeView/LoadDataOnDemand/description.md

```diff
@@ -1 +1 @@
-In&nbsp;those special cases when the built-in Virtual Mode fails, you can gain full control over data loading in&nbsp;the **createChildren** function.This function is&nbsp;called each time a&nbsp;user expands the TreeView node that has not been expanded before. In&nbsp;the **createChildren** function, you get data from a&nbsp;server or&nbsp;other sources and then form the collection of&nbsp;child nodes from this data.
\ No newline at end of file
+If [virtual mode](/Demos/WidgetsGallery/Demo/TreeView/VirtualMode/) does not suit your scenario, you can use the [createChildren](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#createChildren) function to specify custom logic to load data. This function is called each time a user expands a node that has not been expanded before. The node's identifier is passed to the function as an argument. Send this identifier to your data service that should return data for child nodes.
```

Comment:
>  Send this identifier to your data service that should return data for child nodes.

Why would I need to do this?

Reply:
`does not suit your scenario` Why would it not 'suit your scenario'?

Reply:
- `that has not been expanded before` -> that was not expanded before
- `Send this identifier to your data service that should return data for child nodes` -> Send this identifier to the data service that should return data for child nodes

Reply:
Perhaps just state: 

```suggestion
You can also use the [createChildren](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#createChildren) function to specify custom logic to load data. This function is called each time a user expands a node that has not been expanded before. The node's identifier is passed to the function as an argument. Send this identifier to your data service that should return data for child nodes.
```

Reply:
Reference to virtual mode should be there. What about the following?
```suggestion
You can use the [createChildren](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#createChildren) function to specify custom logic to load data. This function is called each time a user expands a node that was not expanded before. The node's identifier is passed to the function as an argument. Send this identifier to the data service that should return data for child nodes.

As an alternative, you can enable [virtual mode](/Demos/WidgetsGallery/Demo/TreeView/VirtualMode/) to use the default load logic.
```